packages/ripple-keypairs/HISTORY.md

```diff
@@ -4,8 +4,15 @@
 ### Breaking Changes
 * Bump typescript to 5.x
 * Remove Node 14 support
+* Remove `crypto` polyfills, `create-hash`, `elliptic`, `hash.js`, and their many dependencies in favor of `@noble/hashes` and `@nobel/curves`
 * Remove `assert` dependency. If you were catching `AssertionError` you need to change to `Error`.
 * Fix `deriveKeypair` ignoring manual decoding algorithm. (Specifying algorithm=`ed25519` in `opts` now works on secrets like `sNa1...`)
+* Remove `bytesToHex` and `hexToBytes`.  They can now be found in `@xrplf/isomorphic/utils`
+* `verifyTransaction` will throw an error if there is no signature
+* Improved key algorithm detection. It will now throw Errors with helpful messages
+
+### Changes
+* Remove `brorand` as a dependency and use `@xrplf/isomorphic` instead.
```

Comment:
Doesn't this count as a breaking change?

Reply:
No because it is used only internally.  The only reason why i not is if people are directly using `brorand` and want to eliminate that dependency by using `@xrpl/isomorphic`.